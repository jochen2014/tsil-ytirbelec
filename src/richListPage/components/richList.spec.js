import React from 'react';
import renderer from 'react-test-renderer';
import RichList from './richList';

describe('RichList Test', ()=>{
    it('should render correctly' ,()=>{
        const data = [
            {
                rank:1,
                name:'bo chen',
                country:'Australia',
                netWorth:210000000
            },
            {
                rank:1,
                name:'test',
                country:'United State',
                netWorth:200000000
            }
        ];
        const tree = renderer.create(<RichList richList={data} currencyCode='usd' />);
        expect(tree.toJSON()).toMatchSnapshot();
    })
})