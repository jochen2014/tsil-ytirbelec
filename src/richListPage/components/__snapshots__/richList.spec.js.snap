// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RichList Test should render correctly 1`] = `
<div
  className="data-list"
>
  <div
    className="data-item"
  >
    <div
      className="rank"
    >
       
      <span>
        No: 
        1
      </span>
    </div>
    <div>
      <span>
        Name: 
        bo chen
      </span>
    </div>
    <div>
      <span>
        Net Worth: 
        $USD 210,000,000.00
      </span>
    </div>
    <div>
      <span>
        Age: 
      </span>
    </div>
    <div>
      <span>
        Country of Birth: 
        Australia
      </span>
    </div>
  </div>
  <div
    className="data-item"
  >
    <div
      className="rank"
    >
       
      <span>
        No: 
        1
      </span>
    </div>
    <div>
      <span>
        Name: 
        test
      </span>
    </div>
    <div>
      <span>
        Net Worth: 
        $USD 200,000,000.00
      </span>
    </div>
    <div>
      <span>
        Age: 
      </span>
    </div>
    <div>
      <span>
        Country of Birth: 
        United State
      </span>
    </div>
  </div>
</div>
`;
